package-lock.json

```diff
@@ -5333,6 +5889,12 @@
       "resolved": "https://registry.npmjs.org/map-cache/-/map-cache-0.2.2.tgz",
       "integrity": "sha1-wyq9C9ZSXZsFFkW7TyasXcmKDb8="
     },
+    "map-obj": {
+      "version": "2.0.0",
+      "resolved": "https://registry.npmjs.org/map-obj/-/map-obj-2.0.0.tgz",
+      "integrity": "sha1-plzSkIepJZi4eRJXpSPgISIqwfk=",
```

Comment:
I've reinstalled node and npm, and then followed the instructions described here: https://github.com/npm/npm/issues/16938#issuecomment-339863980

The diffs that change the integrity hash of existing packages are gone, but there still seems to be some sha1 hashes used for new packages. Is this expected? (I took a look at the `package-lock.json` file on master, and it seems to have some sha1 hashes as well)

Reply:
The sha1 hashes should be expected. I tried on your new `package-lock.json`, and there were no changes now. :+1: